client/src/pages/Home.jsx

```diff
@@ -26,9 +28,17 @@ const Home = () => {
     setCoordinates({ lat: e.latLng.lat(), lng: e.latLng.lng() });
   };
 
-  const updateCoordinates = () =>{
+  function updateCoordinates(e){
     setCoordinates(markerPosition);
     setShowCoordinates(true);
+
+
+    e.preventDefault();
+    axios.post('/make_guess',coordinates)
+    .then((res)=>{
+      console.log(res)
+    })
+    .catch((err)=>console.log(err))
```

Comment:
No actions required for now. But we should think about hwo to properly handle errors, as well as sending the correct status codes (e.g., 200, 401, etc.)